### PR TITLE
D1 wizard documentation

### PR DESCRIPTION
**Pull request by MiquelIR**
### Description:

> Add documentatio for the GiscedataChangeToRemoteManagedWithCCH wizard

### Tasks:

- [x] Add docs
- [x] Translate

### Links:

> Add links for each file modified

* [polisses.md Assistent de canvi a operativa amb cch](http://builds.gisce.net/powerp-docs/powerp_D1_wizard_documentation/ca/distri/polizas/#assistent-de-canvi-a-operativa-amb-cch)
* [polizas.md Asistente de camcio a operativa con cch](http://builds.gisce.net/powerp-docs/powerp_D1_wizard_documentation/es/distri/polizas/#asistente-de-cambio-a-operativa-con-cch)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_D1_wizard_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_D1_wizard_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/polizas.md [hunk 1]
```diff
@@ -149,3 +149,69 @@ provoquen. Els errors poden ser els següents:
 ![](_static/polizas/error_1.png)
 
 ![](_static/polizas/error_2.png)
+
+## Assistent de canvi a Operativa amb CCH
+
+### Funcionament
```
Comment (jaumef) [inline docs/distri/polizas.md]: Aquest títol es pot obviar i ens emplena els títols

File: docs/distri/polizas.md [hunk 2]
```diff
@@ -149,3 +149,69 @@ provoquen. Els errors poden ser els següents:
...
+    es té **facturat tot el període anterior**.
+
+!!! Info "Nota"
+    És possible utilitzar l'assistent per **múltiples contractes** al mateix temps. Per
+    fer-ho haurem de seleccionar de llistat de contractes tots els que ens interessa
+    modificar i executar l'assistent.
+
+#### Pas 1
+
+##### Opcions
```
Comment (jaumef) [inline docs/distri/polizas.md]: Ho pots deixar amb negreta únicament així no emplenem l'apartat de subtítols?

File: docs/distri/polizas.md (hunk 1 above)
Comment (MiquelIR) [reply docs/distri/polizas.md]: =(

File: docs/distri/polizas.md (hunk 2 above)
Comment (MiquelIR) [reply docs/distri/polizas.md]: =(

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_D1_wizard_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**